pages/doc/integrations_aws_metrics.md

```diff
@@ -301,14 +302,22 @@ Unless otherwise indicated, VMware Aria Operations for Applications sets the val
   - `aws.rds.capacity` - For Amazon Aurora only, RDS capacity.
   - `aws.rds.backtrackconsumedchangerecords` - For Amazon Aurora only, the number of change records stored for Backtrack.
 
-- Service Limit Metrics - Capture the current resource limits and usage for your AWS account. These metrics include the point tags `Region` and `category`.
+- (Optional) Service Limit Metrics - Capture the current resource limits and usage for your AWS account. These metrics include the point tags `Region` and `category`.
   - `aws.limits.<resource>.limit` - The current limit for an AWS resource in a particular region.
   - `aws.limits.<resource>.usage` - The current usage of an AWS resource in a particular region.
 
-    {% include note.html content="To examine these metrics, your account needs at least the Business-level AWS Support plan because the integration uses the Support API to pull service limits. You also need both ReadOnlyAccess and AWSSupportAccess. See [Giving VMware Aria Operations for Applications Read-Only Access](integrations_aws_overview.html#giving-access-to-your-aws-account) for details." %}
+    {% include note.html content="You can [disable](#disable-service-limit-metrics-ingestion) the ingestion of these metrics. If these metrics are enabled, to examine them, your account needs at least the Business-level AWS Support plan, because the integration uses the Support API to pull service limits. You also need both ReadOnlyAccess and AWSSupportAccess. See [Giving VMware Aria Operations for Applications Read-Only Access](integrations_aws_overview.html#giving-access-to-your-aws-account) for details." %}
     
+### Disable Service Limit Metrics Ingestion
 
+After you [set up an AWS integration](integrations_aws_overview.html#set-up-an-aws-integration) associated it with a Role ARN, you can disable the ingestion of support service limit metrics (also known as service quotas), that is, the metrics namespace `aws.limits.*`. You can later enable these metrics again, if necessary.
 
+1. Log in to your product cluster and click **Integrations** on the toolbar.
+1. In the Featured section, click the **Amazon Web Services** tile.
+1. Click the **Setup** tab.
+1. In the Types column, click the **AWS Metrics+** link in the row of the integration you want to configure.
+1. Deselect the **Service Limit Metrics** checkbox.
```

Comment:
1. In the Types column, of the row of the integration you want to configure, click the **AWS Metrics+** link.
1. On the **Edit AWS Metrics+** page, deselect the **Service Limit Metrics** check box.